.travis.yml

```diff
@@ -4,8 +4,23 @@ go:
   - 1.8
 
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
From the travis docs, it looks like we can set sudo to false and just get the docker binaries. 

Docker is installed. We use the official Docker apt repository, so you can easily install another version with apt if needed. When sudo: required is specified, Docker is installed as a service. When sudo: false is specified, the Docker binaries are available for local use.